site.yml

```diff
@@ -38,5 +38,5 @@
 
 - hosts: veyepar-encoder
   roles:
-  - nfs-client
+      # - nfs-client
```

Comment:
I only 1/2 agree.  

If I had a good grip on all the scenarios I wanted to handle, then I can see config flags.  but over the years and venues various ways of getting access to the files come and go. 

I suspect the people using this stuff will want to tailor things based on available resources, which means editing these files, so putting a config somewhere else makes the ad hoc stuff harder.

plus it is quicker and I am not yet sure what the conventions are for adding configs.